### PR TITLE
CORS-snapshot-issue-fix

### PR DESCRIPTION
**Pull request by tytarman**
# **Pull Request Template**

###### *Select a category that relates to the content of your pull request (choose all that apply).*

- [x] Bug Fix
- [ ] Content
- [ ] New Feature
- [ ] Documentation
- [ ] Code Refactoring

&nbsp;

###### *Provide a brief summary of the changes you made and their purpose as they relate to Index UI. <br/>If applicable, please link the corresponding issue number.*

The snapshot page was throwing a CORS error in the console. I updated the code to resolve the issue and tested in Brave, Firefox, and Safari on my MacOS laptop.

### **Fixes: Issue #365**

&nbsp;

###### *Include proof that your changes function properly and resolve the linked issue.*

The console still threw the Google Tag Manager warning and Firefox threw an additional cookie warning; however the CORS issue was resolved by reworking some of the Snapshot data.

&nbsp;

CORS issue (Brave)
CORS issue (Firefox)
CORS issue (Safari)

###### *By submitting this pull request, you are confirming the following to be true:*

- I have reviewed the [Contribution Guidelines](https://github.com/SetProtocol/index-ui/blob/master/CONTRIBUTING.md).
- I have performed a self-review of my own code.
- I have updated my repository to match the master at `SetProtocol/index-ui`.
- I have included test data or screenshots that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by tytarman** [comment]
@0xModene Let me know if there's anything else I need to do with it

**Review by 0xModene — APPROVED** [review]
lgtm!

File: src/App.tsx [hunk 1]
```diff
@@ -170,11 +173,9 @@ const Providers: React.FC = ({ children }) => {
                                                     <BedTokenMarketDataProvider>
                                                       <BedIndexComponentsProvider>
                                                         <IndexTokenMarketDataProvider>
-                                                          <SnapshotProposalsProvider>
```
Comment (rootulp) [inline src/App.tsx:173]: [question] does it make sense to delete the `<SnapshotProposalsProvider>` given we're no longer using it?

**Review by rootulp — COMMENTED** [review]
Thanks for resolving the issue!

File: src/App.tsx (hunk 1 above)
Comment (0xModene) [reply src/App.tsx:173]: yes, @tytarman actually deleted it here

Comment (rootulp) [reply src/App.tsx:173]: Sorry for not being, clear. I'm proposing deleting https://github.com/SetProtocol/index-ui/blob/cfcdcbc23966e2afa13dc0868a72eb78e7f247f8/src/contexts/SnapshotProposals/SnapshotProposalsProvider.tsx#L5-L25 and any other types, interfaces, files that are no longer used.

Comment (0xModene) [reply src/App.tsx:173]: 🤦 of course I misunderstood that, yes I think we should. I'll make an issue for it

**Pull request merged**